src/main/java/Task.java

```diff
@@ -0,0 +1,60 @@
+public class Task {
+
+    private int id;
+    private String taskInfo;
+    private boolean isDone;
+
+    private static int numberOfTasks = 0;
+
+    public Task(String taskInfo) {
+        setTaskInfo(taskInfo);
+        numberOfTasks++;
+        setId(numberOfTasks);
+        this.isDone = false;
+    }
+
+    public void markAsDone() {
```

Comment:
Can consider combining the markAsDone and markAsUndone function if you want by editing the earlier changeTaskStatus function.